staging/src/k8s.io/apiextensions-apiserver/pkg/controller/openapi/builder/builder.go

```diff
@@ -65,6 +70,14 @@ var (
 	namespaceToken = "{namespace}"
 )
 
+// The path for definitions in OpenAPI v2 and v3 are different. Translate the path if necessary
+func translateRefIfNeeded(schemaRef string, v2 bool) string {
```

Comment:
refForOpenAPIVersion would be a better name.

Reply:
rule of thumb: use function names that don't force you to look into the function implementation to know what it does